src/templates/html/map/Map.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import { any, string } from "zod";
```

Comment:
Is this correct or an accidental import?

Reply:
accidental - will remove - good catch !